### PR TITLE
networking: Convert switchbox() to React 

### PR DESCRIPTION
**Pull request by mvollmer**

### CODE REVIEW

**Comment by mvollmer** [comment]
@martinpitt, afaics, OnOff components can only be toggled and it doesn't matter whether you click on the "Off" or "On" part.  The difference between the old button and the new one is only that the new one needs the " span" at the end of the selector.

File: test/verify/netlib.py [hunk 1]
```diff
@@ -116,3 +116,13 @@ def slow_down_dhclient(self, delay):
...
+
+    def wait_onoff(self, sel, val):
+        self.browser.wait_present(sel + " .btn.active:contains('%s')" % ("On" if val else "Off"))
+
+    def toggle_onoff(self, sel):
+        # PR #11773 moves to our React component, which changes what we have to click on
+        if self.machine.image in ["rhel-7-6-distropkg", "rhel-8-0-distropkg"]:
+            self.browser.click(sel + " .btn:contains('On')")
+        else:
+            self.browser.click(sel + " .btn:contains('On') span")
```
Comment (martinpitt) [inline test/verify/netlib.py]: The extra "span" doesn't hurt the old tests, so if always clicking on "on" actually works, this special case can just go away completely. I just thought that for the React component you always have to click on the non-current half of it, but maybe/hopefully that was wrong then.

**Comment by martinpitt** [comment]
And all tests fail with
```
 File "/build/cockpit/bots/../test/verify/check-networking-firewall", line 62
    self.wait_onoff("#networking-firewall-switch", False)
```

**Comment by mvollmer** [comment]
> And all tests fail with

Ouch, fixed.

File: test/verify/netlib.py (hunk 1 above)
Comment (mvollmer) [reply test/verify/netlib.py]: Let's try that!

**Comment by martinpitt** [comment]
```
    self.wait_onoff("#networking-firewall-switch", False)
AttributeError: 'TestFirewall' object has no attribute 'wait_onoff'
```

**Comment by mvollmer** [comment]
Okay, new try.  I can't say I am on top of all the combinations... One thing I was wrong about is the toggle behavior of the old buttons.

**Comment by martinpitt** [comment]
```
    if m.image in ["rhel-7-6-distropkg", "rhel-8-0-distropkg"]:
NameError: name 'm' is not defined
```

**Comment by martinpitt** [comment]
Force-pushed to fix the issue above, and add missing Closes #. Locally ran -team test, works fine now.

**Comment by mvollmer** [comment]
> Force-pushed to fix the issue above, and add missing Closes #.

Thanks!

**Pull request merged**